05/app.js

```diff
@@ -46,10 +98,19 @@ if (formEl !== null) {
 			displayErrorMessage("Proszę wybrać województwo.");
 		}
 
-		if (validateZipCode(zipCode) && validateVoivodeship(voivodeship)) {
+		if (
+			validateZipCode(zipCode) &&
+			validateVoivodeship(voivodeship) &&
+			validateNameAndSurname(firstName, lastName) &&
+			validateStreetAndFlatNumbers(streetNumber, flatNumber, houseNumber) &&
+			validateCity(cityName)
```

Comment:
Aby kod wew. `if` został wykonany to warunek musi być prawdziwy.
W tym przypadku, aby był prawdziwy to każda z funkcji musi zwrócić `true` - trzeba sprawdzić czy tak faktycznie jest.

Więc sprawdzam po kolei każdą funkcję...